agent_plugin_builder/vendor_dirs.py

```diff
@@ -1,54 +1,81 @@
+import logging
 from pathlib import Path
-
-import subprocess
-import docker
+from shlex import quote
+from typing import Final
 
 from monkeytypes import OperatingSystem
 
+import docker
+
 from .compare_package_lists import all_equal, load_package_names
 
-AGENT_PLUGIN_IMAGE = "infectionmonkey/agent-builder:latest"
-PLUGIN_BUILDER_IMAGE = "infectionmonkey/plugin-builder:latest"
-LINUX_IMAGE_PYENV_INIT_COMMANDS = [
+logger = logging.getLogger(__name__)
+
+
+AGENT_PLUGIN_IMAGE: Final = "infectionmonkey/agent-builder:latest"
+PLUGIN_BUILDER_IMAGE: Final = "infectionmonkey/plugin-builder:latest"
```

Comment:
It might be better if theses were named "LINUX_PLUGIN_BUILDER_IMAGE" and "WINDOWS_PLUGIN_BUILDER_IMAGE". Since that's how they're used, the names would better capture their intent. Plus, it may not always be the case that Linux plugins are built with the agent-builder image.